docs/releases/4.0.txt

```diff
@@ -441,6 +441,8 @@ Miscellaneous
 * The undocumented ``django.utils.baseconv`` module is deprecated.
 
 * The undocumented ``django.utils.datetime_safe`` module is deprecated.
+* The default sitemap 'protocol' will be changed from 'http' to 'https'
```

Comment:
Needs a blank line. But this should go in the `contrib.sitemaps` section (around line 132 at this revision.)